adapters/smaato/smaatotest/supplemental/bad-user-ext-data-req.json

```diff
@@ -0,0 +1,126 @@
+{
+  "mockBidRequest": {
```

Comment:
There are 2 mockBidRequest values in this request and it's not clear which one is being used. Can you please remove the duplicate?

Reply:
done